debian/bunsen-configs-wbase.install

```diff
@@ -0,0 +1,18 @@
+bin usr
+applications usr/share
+geany usr/share
+skel usr/share/bunsen
```

Comment:
@johnraff there seems to be a _lot_ of stuff here that doesn't need to be in the base install, let alone the `wbase`

```
mick@dellhome:~/misc/data/usr/share/bunsen/skel/.config$ tree
.
├── autostart
│   └── xbindkeys.desktop
├── blob
│   └── current
├── bunsen
│   ├── autostart
│   └── environment
├── compton.conf
├── conky
│   ├── BL-Boron-conky.conf
│   └── BL-Default-conky.conf -> BL-Boron-conky.conf
├── dmenu
│   └── dmenu-bind.sh
├── fontconfig
│   └── fonts.conf
├── geany
│   ├── colorschemes
│   ├── filedefs
│   │   ├── filetypes.common
│   │   └── filetypes.README
│   ├── geany.conf
│   ├── plugins
│   ├── tags
│   ├── templates
│   │   ├── files
│   │   └── templates.README
│   └── ui_toolbar.xml
├── gigolo
│   └── config
├── gsimplecal
│   └── config
├── gtk-3.0
│   ├── bookmarks.template
│   ├── gtk.css
│   └── settings.ini
├── jgmenu
│   ├── hooks
│   ├── jgmenurc
│   ├── prepend.csv
│   └── startup
├── lxterminal
│   └── lxterminal.conf
├── mimeapps.list
├── nitrogen
│   ├── bg-saved.cfg
│   └── nitrogen.cfg.template
├── openbox
│   ├── autostart
│   ├── bl-menu.xml
│   └── bl-rc.xml
├── picom.conf
├── pnmixer
│   └── config
├── terminator
│   └── config
├── Thunar
│   ├── accels.scm
│   └── uca.xml
├── tint2
│   ├── beryllium.tint2rc
│   ├── boron-dark-horizontal.tint2rc
│   ├── boron-dark-vertical.tint2rc
│   ├── boron-light-horizontal.tint2rc
│   ├── boron-light-vertical.tint2rc
│   ├── crunchbang.tint2rc
│   ├── fever_room.tint2rc
│   ├── grey.tint2rc
│   ├── helium.tint2rc
│   ├── hidpi.tint2rc
│   ├── lithium-light.tint2rc
│   ├── lithium-light-vertical.tint2rc
│   ├── lithium.tint2rc
│   ├── lithium-vertical.tint2rc
│   ├── tint2rc -> boron-dark-vertical.tint2rc
│   └── yeti.tint2rc
├── Trolltech.conf
├── user-dirs.dirs
├── user-dirs.locale
└── xfce4
    ├── helpers.rc
    └── xfconf
        └── xfce-perchannel-xml
            ├── thunar-volman.xml
            ├── thunar.xml
            ├── xfce4-appfinder.xml
            ├── xfce4-notifyd.xml
            ├── xfce4-panel.xml
            └── xfce4-power-manager.xml

28 directories, 61 files
```

Reply:
That is the `.config/` tree from `bunsen-configs-base_13.0-1_all.deb`

Reply:
That's all config stuff, I don't think there's anything there that would do any harm - please say if you spot something. It's there because of the way the "base" and "lite" configs were added on top of the original configs, overwriting any files that need to be different. You'll see that the base and lite trees are quite small.

I did it that way because I didn't want to maintain two (at the time, but now three) separate file trees with almost identical content. It would have given much more scope for mistakes, and anyway would have been more work.

Reply:
I suppose it cant hurt to be there.

Still, I want to tidy some things up before I mark as ready to merge.